solarforecastarbiter/metrics/calculator.py

```diff
@@ -110,6 +117,13 @@ def _apply_probabilistic_metric_func(metric, fx, fx_prob, obs, **kwargs):
         return metric_func(obs, fx, fx_prob)
 
 
+def _apply_event_metric_func(metric, fx, obs, **kwargs):
+    """Helper function to deal with variable number of arguments possible for
+    event metric functions."""
+    metric_func = event._MAP[metric][0]
+    return metric_func(obs, fx.to_numpy())
```

Comment:
why `fx.to_numpy()` here and not in other places? comment might be needed.

Reply:
No particular reason; I copied that from the probabilistic code added by Adam and forgot to edit. I'm not 100% sure about the probabilistic code, but for the event data, the `to_numpy()` is not necessary to run. So I'll remove it.